docs/en/latest/plugins/skywalking.md

```diff
@@ -133,7 +133,7 @@ plugin_attr:
         3. Run skywalking server:
 
             ```shell
-            sudo docker run --name skywalking -d -p 1234:1234 -p 11800:11800 -p 12800:12800 --restart always --link elasticsearch:elasticsearch -e SW_STORAGE=elasticsearch -e SW_STORAGE_ES_CLUSTER_NODES=elasticsearch:9200 apache/skywalking-oap-server
+            sudo docker run --name skywalking -d -p 1234:1234 -p 11800:11800 -p 12800:12800 --restart always --link elasticsearch:elasticsearch -e SW_STORAGE=elasticsearch -e SW_STORAGE_ES_CLUSTER_NODES=elasticsearch:9200 apache/skywalking-oap-server:8.3.0-es7
```

Comment:
Should be `8.3.0-es6`?

Reply:
I think there should be `8.3.0-es7`.

I got it from https://github.com/apache/skywalking-docker/blob/master/oap-server/README.md

Or we can remove the doc about `Of Course, you can use Elasticsearch storage`, just link to the doc of SkyWalking.

what do you think @moonming ?

Reply:
yes, you are right. update it later

Reply:
fixed